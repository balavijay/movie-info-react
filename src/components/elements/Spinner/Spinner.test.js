import React from 'react';
import { shallow } from 'enzyme';


import Spinner from './Spinner';

describe ("Spinner Component " , () => {
   const  app = shallow(<Spinner />);
    test("Spinner render" , () => {
        expect(app).toMatchSnapshot();
    })

 
});