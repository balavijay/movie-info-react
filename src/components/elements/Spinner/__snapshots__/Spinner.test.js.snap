// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component  Spinner render 1`] = `ShallowWrapper {}`;
